cxx-sensors/src/main/java/org/sonar/cxx/sensors/coverage/CxxCoverageSensor.java

```diff
@@ -72,42 +90,76 @@ public CxxCoverageSensor(CxxCoverageCache cache, CxxLanguage language) {
   public void describe(SensorDescriptor descriptor) {
     descriptor.onlyOnLanguage(this.language.getKey()).name(language.getName() + " CoverageSensor");
   }
-  
+
   /**
    * {@inheritDoc}
    */
-  public void execute(SensorContext context, Map<InputFile, Set<Integer>> linesOfCode) {
+  @Override
+  public void execute(SensorContext context) {
+    // do nothing
+  }
 
+  /**
+   * {@inheritDoc}
+   * @param context for coverage analysis
+   * @param linesOfCodeByFile use for FORCE_ZERO_COVERAGE_KEY feature 
+   */
+  public void execute(SensorContext context, Map<InputFile, Set<Integer>> linesOfCodeByFile) {
+    String reportsKey = this.language.getStringOption(reportPathKey());
+    LOG.info("Searching coverage reports by path with basedir '{}' and search prop '{}'", 
+        context.fileSystem().baseDir(), reportPathKey());
+    LOG.info("Searching for coverage reports '{}'", reportsKey);
+    
     Map<String, CoverageMeasures> coverageMeasures = null;
     Map<String, CoverageMeasures> itCoverageMeasures = null;
     Map<String, CoverageMeasures> overallCoverageMeasures = null;
 
-    LOG.debug("Coverage BaseDir '{}' ", context.fileSystem().baseDir());    
-    
-    if (this.language.hasKey(REPORT_PATH_KEY)) {
+    LOG.info("Coverage BaseDir '{}' ", context.fileSystem().baseDir());
+
+    if (isSQ_6_2_or_newer) {
+      if (context.settings().hasKey(FORCE_ZERO_COVERAGE_KEY)) {
+        LOG.warn("Property '{}' is deprecated and its value will be ignored.", FORCE_ZERO_COVERAGE_KEY);
```

Comment:
Would prefer to do this at one central place in the plugin:
- xxx no more supported use yyy instead
- xxx is deprecated and its value will be ignored